draft-barnes-acme.md

```diff
@@ -344,12 +347,16 @@ provided by the client.
 The server creates a registration object with the included contact information.  The "key" element of the registration is set to the public key used to verify the JWS (i.e., the "jwk" element of the JWS header).  The server also provides a random
 recovery token.  The server returns this registration object in a 201 (Created) response, with the registration URI in a Location header field.  The server may also indicate its new-authorization URI using the "next" link relation.
 
+If the server wishes to present the client with terms under which the ACME service is to be used, it may indicate the URI where such terms can be accessed in a Link header with link relation "terms-of-service".  As noted above, the client may indicate its
+agreement with these terms by updating its registration to include the "agreement" field, with the terms URI as its value.
+
 ~~~~~~~~~~
 
 HTTP/1.1 201 Created
 Content-Type: application/json
 Location: https://example.com/reg/asdf
 Link: <https://example.com/acme/new-authz>;rel="next"
+Link: <https://example.com/acme/terms>;rel="terms-of-service"
```

Comment:
Why is "terms-of-service" not sufficient?  It's already defined, and seems to fit well enough.  I'm going to go ahead and merge, and we can handle this separately if we need to.